qf_lib/common/utils/error_handling.py

```diff
@@ -55,4 +55,22 @@ def wrap(wrapped_cls):
 
     @classmethod
     def get_error_messages(cls):
+        """
```

Comment:
Method used to get the error messages

        Returns
        -------
        <here type - probably typing.List in this case>
                    current messages that are stored in the class

Reply:
Done